runtime/vm/ContinuationHelpers.cpp

```diff
@@ -82,6 +82,10 @@ createContinuation(J9VMThread *currentThread, j9object_t continuationObject)
 
 	J9VMJDKINTERNALVMCONTINUATION_SET_VMREF(currentThread, continuationObject, continuation);
 
+#ifdef PPC64
+	continuation->jitGPRs.jitGPRs[15] = vm->jitTOC;
```

Comment:
I think it would be worth adding a comment here explaining why this is necessary (the fact that we do not go through the cInterpreter entrypoint).